### ISSUE TITLE
Last release on OpenDDS.org should be 3.18, not 3.16.1

### ISSUE THREAD

**Issue by jwillemsen** [objectcomputing/OpenDDS#2970]
On OpenDDS.org the last release is listed as 3.16.1, but that should be 3.18, 3.16.1 seems to be a bugfix release which should be in the list with previous releases but the high lighted release should be 3.18 to my idea

**Comment by mitza-oci** [comment]
Right, I think @iguessthislldo and @ClaytonCalabrese were looking at this yesterday.  Any thoughts on how to fix it?

**Comment by iguessthislldo** [comment]
We could replace [the ` sort: "created_at" | reverse` part of this template expresssion](https://github.com/objectcomputing/OpenDDS/blob/bfebbad6ffdd182025880c16eda436ad314c1e9a/_includes/release_summary.html#L4) with something that can sort by the version numbers. I'd actually like it to also omit older patch releases, so that if we release 3.18.1, then 3.18.0 isn't on the list. That way someone on an older release has a better change of seeing newer patch releases for their minor version. I'm not very familiar with Jekyll, but if it's similar to Django or Jinja templates, then we might have to write some ruby to get it to do what we want because sorting by version probably won't be builtin.

**Comment by iguessthislldo** [comment]
Tried to get this working, but [it looks like GitHub prevents the running of unapproved Jekyll plugins](https://jekyllrb.com/docs/plugins/installation/). This means we have to either run Jekyll ourselves each time we want to publish or figure out how to do it with the builtin template language and maybe Javascript.
